src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -628,18 +628,11 @@ protected DynamoModel(IStartConfiguration config)
 
             UpdateManager = config.UpdateManager ?? new DefaultUpdateManager(null);
```

Comment:
any reason to still even construct this object?

Reply:
@mjkkirschner I think just a chain of code depending on this object to exist but currently it is not guaranteed. e,g, Dynamo for C3D does not initialize it and that is causing some bugs